micropython/drivers/radio/nrf24l01/nrf24l01test.py

```diff
@@ -7,19 +7,21 @@
 from nrf24l01 import NRF24L01
 from micropython import const
 
-# Slave pause between receiving data and checking for further packets.
+# Responder pause between receiving data and checking for further packets.
 _RX_POLL_DELAY = const(15)
-# Slave pauses an additional _SLAVE_SEND_DELAY ms after receiving data and before
-# transmitting to allow the (remote) master time to get into receive mode. The
-# master may be a slow device. Value tested with Pyboard, ESP32 and ESP8266.
-_SLAVE_SEND_DELAY = const(10)
+# Responder pauses an additional _RESPONER_SEND_DELAY ms after receiving data and before
+# transmitting to allow the (remote) initiator time to get into receive mode. The
+# initiator may be a slow device. Value tested with Pyboard, ESP32 and ESP8266.
+_RESPONDER_SEND_DELAY = const(10)
 
 if usys.platform == "pyboard":
     cfg = {"spi": 2, "miso": "Y7", "mosi": "Y8", "sck": "Y6", "csn": "Y5", "ce": "Y4"}
 elif usys.platform == "esp8266":  # Hardware SPI
     cfg = {"spi": 1, "miso": 12, "mosi": 13, "sck": 14, "csn": 4, "ce": 5}
 elif usys.platform == "esp32":  # Software SPI
     cfg = {"spi": -1, "miso": 32, "mosi": 33, "sck": 25, "csn": 26, "ce": 27}
+elif usys.platform == "rp2":  # Hardware SPI with explicit pin definitions
+    cfg = {"spi": -2, "miso": 4, "mosi": 3, "sck": 2, "csn": 5, "ce": 6}
```

Comment:
Would it be simpler to make this a dict that can be passed directly to the SPI constructor, like `SPI(**cfg)`?

Or even simpler, just create the SPI object here, eg `spi = SPI(0, sck=Pin(2), ...)`?

Reply:
> Or even simpler, just create the SPI object here, eg spi = SPI(0, sck=Pin(2), ...)?

I agree - I was aiming to minimise changes. We would still need `mosi`, `miso` and `sck` in the dict because of the print statements at the end.

Do you want to make this change for each supported platform?

Reply:
Yes, please make the change for each platform, if you have the time.

Also, instead of printing the SPI pins at the end, it could just print the `spi` object, which in some cases will show the pins anyway (also baudrate etc).